tests/CommonTestUtils/GitModuleTestHelper.cs

```diff
@@ -208,8 +239,27 @@ public void Dispose()
                 }
             }
 
+#if !CI_BUILD
+            TestCleanUp.RegisterCleanUpAction(
+                () =>
+                {
+                    CleanUpOperations.FileAndForget(
+                        () => CleanUp(TemporaryPath));
+                });
```

Comment:
I would prefer to have it on one line although I use to request Left Reading style.
At least, `() =>` is not worth a separate line (common style in the code base).

Reply:
Huh, okay 🙂 I find it easier to reason about code when it's set out like this. Easy to change.